unittest/test_diskann.cpp

```diff
@@ -316,10 +336,39 @@ TEST_P(DiskANNTest, knn_search_test) {
 
     if (metric_ == knowhere::metric::IP) {
         auto ip_recall = CheckTopKRecall(ip_ground_truth_, ids, kK);
-        EXPECT_GT(ip_recall, 0.8);
+        EXPECT_GT(ip_recall, 0.7);
```

Comment:
Aha, what happened

Reply:
Nothing, just for safety. I lowered build parameters dramatically, but it will still pass even set to 0.8.

Reply:
Ahahaha, OK, it is fine to me